utils/Variables.js

```diff
@@ -0,0 +1,5 @@
+const uploadsUrl = 'https://media.mw.metropolia.fi/wbma/uploads/';
+const apiUrl = 'https://media.mw.metropolia.fi/wbma/';
+const appID = 'KRITICS_TEMP_ID';
```

Comment:
Do you think that we need to embed the appID in our initial values for sign in/sign up, so that only the users who sign up at our page can log in to their account. Otherwise, anyone who signs up in any other app can also sign in to our app because we share the database.